src/marketplace/sellers/mftf-vendor.md

```diff
@@ -0,0 +1,75 @@
+---
```

Comment:
As this file is identical to the first file, please apply all corrections show above to this file.